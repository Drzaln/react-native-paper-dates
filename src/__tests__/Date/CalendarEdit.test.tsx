import { render } from '@testing-library/react-native'
import CalendarEdit from '../../Date/CalendarEdit'

it('renders CalendarEdit', () => {
  const { toJSON } = render(
    <CalendarEdit
      mode="single"
      state={{
        startDate: new Date(),
        endDate: new Date(),
        date: new Date(),
        dates: [new Date()],
      }}
      collapsed={false}
      onChange={() => null}
      validRange={undefined}
      locale="en"
    />
  )
  expect(toJSON()).toMatchSnapshot()
})
