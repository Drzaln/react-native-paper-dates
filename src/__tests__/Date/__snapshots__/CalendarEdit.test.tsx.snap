// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CalendarEdit 1`] = `null`;
